app/src/main/res/layout/exploration_activity.xml

```diff
@@ -55,6 +55,14 @@
             android:scaleType="center"
             app:srcCompat="@{viewModel.isAudioStreamingOn ? @drawable/ic_audio_streaming_on_24dp : @drawable/ic_audio_streaming_off_24dp}"
             android:visibility="@{viewModel.showAudioButton ? View.VISIBLE : View.GONE}" />
+
+          <ImageButton
+            android:id="@+id/action_bottom_sheet"
```

Comment:
IDs are global to the app--this should be more specific to what specific menu it corresponds to.

Ditto for the options in the bottom sheet fragment layout below.